cmd/translate/main_grpc_integration_test.go

```diff
@@ -201,3 +204,175 @@ func Test_DownloadTranslationFile_gRPC(t *testing.T) {
 		})
 	}
 }
+
+// ------------------Service------------------
+
+func randService() *translatev1.Service {
+	return &translatev1.Service{
+		Id:   gofakeit.UUID(),
+		Name: gofakeit.FirstName(),
+	}
+}
+
+func Test_CreateService_gRPC(t *testing.T) {
+	t.Parallel()
+
+	service := randService()
+
+	_, err := client.CreateService(context.Background(), &translatev1.CreateServiceRequest{Service: service})
+	if !assert.NoError(t, err, "Prepare test data") {
+		return
+	}
+
+	actual := status.Code(err)
+	expected := codes.OK
+
+	assert.Equal(t, expected, actual, "want codes.%s got codes.%s", expected, actual)
+}
+
+func Test_UpdateServiceAllFields_gRPC(t *testing.T) {
+	t.Parallel()
+
+	ctx := context.Background()
+
+	service := randService()
+
+	_, err := client.CreateService(ctx, &translatev1.CreateServiceRequest{Service: service})
+	if !assert.NoError(t, err, "Prepare test data") {
+		return
+	}
+
+	service.Name = gofakeit.FirstName()
+
+	_, err = client.UpdateService(ctx, &translatev1.UpdateServiceRequest{
+		Service:    service,
+		UpdateMask: nil,
+	})
+	if !assert.NoError(t, err) {
+		return
+	}
+
+	actual := status.Code(err)
+	expected := codes.OK
+
+	assert.Equal(t, expected, actual, "want codes.%s got codes.%s", expected, actual)
+}
+
+func Test_UpdateServiceSpecificField_gRPC(t *testing.T) {
+	t.Parallel()
+
+	ctx := context.Background()
+
+	service := randService()
+
+	_, err := client.CreateService(ctx, &translatev1.CreateServiceRequest{Service: service})
+	if !assert.NoError(t, err, "Prepare test data") {
+		return
+	}
+
+	service.Name = gofakeit.FirstName()
+
+	_, err = client.UpdateService(ctx, &translatev1.UpdateServiceRequest{
+		Service:    service,
+		UpdateMask: &fieldmaskpb.FieldMask{Paths: []string{"name"}},
```

Comment:
When adding FieldMask, should it be `service.name`, or just `service` ?

Reply:
for primary resource "name", for nested subresource.name